dotCMS/src/integration-test/java/com/dotcms/enterprise/publishing/remote/DependencyBundlerTest.java

```diff
@@ -0,0 +1,572 @@
+        //Create contentlet
+        final Contentlet contentlet = TestDataUtils.getWikiContent(true,APILocator.getLanguageAPI().getDefaultLanguage().getId(),contentType.id());
+        //Create Folder
+        final Folder folder = new FolderDataGen().nextPersisted();
+        //Create Page
+        final HTMLPageAsset page = new HTMLPageDataGen(folder,template).languageId(APILocator.getLanguageAPI().getDefaultLanguage().getId())
+                .nextPersisted();
+        HTMLPageDataGen.publish(page);
+        //Add Contentlet to Page
+        final MultiTree multiTree = new MultiTree(page.getIdentifier(),
+                container.getIdentifier(),
+                contentlet.getIdentifier(), getDotParserContainerUUID(uuid), 0);
+        APILocator.getMultiTreeAPI().saveMultiTree(multiTree);
+
+        //Create bundle with DefaultFilter
+        final Bundle bundleWithDefaultFilter = createBundle("TestBundle"+System.currentTimeMillis(),false,defaultFilterKey);
+        //Add assets to the bundle
+        PublisherAPI.getInstance().saveBundleAssets(Arrays.asList(page.getIdentifier()),bundleWithDefaultFilter.getId(),
+                systemUser);
+        //Generate Bundle, will return several dependencySet with the assets that will be added to the bundle
+        final PushPublisherConfig listOfAssetsWithDefaultFilter = generateBundle(bundleWithDefaultFilter.getId(), Operation.PUBLISH);
+        Assert.assertNotNull(listOfAssetsWithDefaultFilter);
+        Assert.assertFalse(listOfAssetsWithDefaultFilter.getStructures().isEmpty());
+        Assert.assertTrue(listOfAssetsWithDefaultFilter.getStructures().contains(contentType.id()));
+        Assert.assertTrue(listOfAssetsWithDefaultFilter.getStructures().contains(page.getContentType().id()));
+        Assert.assertFalse(listOfAssetsWithDefaultFilter.getTemplates().isEmpty());
+        Assert.assertTrue(listOfAssetsWithDefaultFilter.getTemplates().contains(template.getIdentifier()));
+        Assert.assertFalse(listOfAssetsWithDefaultFilter.getContainers().isEmpty());
+        Assert.assertTrue(listOfAssetsWithDefaultFilter.getContainers().contains(container.getIdentifier()));
+        Assert.assertFalse(listOfAssetsWithDefaultFilter.getContentlets().isEmpty());
+        Assert.assertTrue(listOfAssetsWithDefaultFilter.getContentlets().contains(contentlet.getIdentifier()));
+        Assert.assertTrue(listOfAssetsWithDefaultFilter.getContentlets().contains(page.getIdentifier()));
+        Assert.assertFalse(listOfAssetsWithDefaultFilter.getFolders().isEmpty());
+        Assert.assertTrue(listOfAssetsWithDefaultFilter.getFolders().contains(folder.getInode()));
+
+
+        //Create filter
+        final String filterKey = "TestFilterWithDependenciesFalse.yml";
+        final List<Object> excludeClassesList = new ArrayList<>();
+        excludeClassesList.add("Template");
+        excludeClassesList.add("Containers");
+        createFilterDescriptor(filterKey,false,true,false,
```

Comment:
the `excludeClassesList` seems unnecessary here

Reply:
done